Cargo.toml

```diff
@@ -17,23 +16,28 @@ members = [
 ]
 
 [workspace.package]
+# Vello version, also used by other packages which want to mimic Vello's version.
+# Right now those packages include vello_encoding and vello_shaders.
+#
+# NOTE: When bumping this, remember to also bump the aforementioned other packages'
+#       version in the dependencies section at the bottom of this file.
```

Comment:
We should also mention the `simple` example's `Cargo.toml`, which should point to the released version in the comment

Reply:
Good call, I tweaked the simple example and added the comment to the workspace `Cargo.toml` as well.